test/unit/@node-red/runtime/lib/nodes/.testUserHome/.config.runtime.json

```diff
@@ -0,0 +1,4 @@
+{
+    "instanceId": "bff8dba3f474d406",
+    "_credentialSecret": "c9ca84d93a4f6236426cc740b147b50f32b3dd9aab9f4191390a604197722a4d"
```

Comment:
Check this file as well as it's another new file that I'm node sure should be checked in

Reply:
again - no idea... something must have changed in the test framework to create those... deleted